bigflow/build/reflect.py

```diff
@@ -152,6 +171,27 @@ def _locate_plain_setuppy_on_fs(stack):
         return None
 
 
+@public()
+def get_project_spec(
+    project_name: Optional[str] = None,
+) -> BigflowProjectSpec:
+    """Read & parses project spec.
+    This function should be used from inside project.
```

Comment:
Could you be more specific here? This is public method but docstring says `should be used from inside project.`

Reply:
I've changed docstring.

Reply:
👍